webserver/views/datasets.py

```diff
@@ -335,6 +332,22 @@ def recording_info(mbid):
         return jsonify(error=str(e)), 404
 
 
+@datasets_bp.route("/metadata/recording/<uuid:mbid>")
+@login_required
+def recording_info(mbid):
+    """Endpoint for getting information about recordings (title and artist)."""
+    return _get_recording_info_for_mbid(mbid)
+
+
+@datasets_bp.route("/metadata/dataset/<uuid:dataset_id>/<uuid:mbid>")
+def recording_info_in_dataset(dataset_id, mbid):
```

Comment:
I think adding the expected behaviour of not returning data for recordings not in the dataset to the docstring here would be a good idea.